infra/aws/terraform/prow-build-cluster/resources/external-secrets/configmaps.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: google-adc
+  namespace: default
```

Comment:
Shouldn't we put this ConfigMap in the `external-secrets` namespace, since as far as I can tell, it's supposed to by used by ESO?

Reply:
fixed